Sources/PostgreSQL/Connection/PostgreSQLConnection+TCP.swift

```diff
@@ -1,50 +0,0 @@
-import Async
-import NIO
-import NIOOpenSSL
-
-extension PostgreSQLConnection {
-    /// Connects to a Redis server using a TCP socket.
-    public static func connect(
-        hostname: String = "localhost",
```

Comment:
There needs to be a deprecated method for this old API

Reply:
Added one.